tools/postman/AWS production.postman_environment.json

```diff
@@ -0,0 +1,29 @@
+{
+	"id": "40e90b8a-5aef-47ce-b53b-b0142ba1ceb9",
+	"name": "AWS production",
+	"values": [
+		{
+			"key": "baseURL",
+			"value": "https://eapd-api.cms.gov",
+			"enabled": true
+		},
+		{
+			"key": "created-api-id",
+			"value": "1",
+			"enabled": true
+		},
+		{
+			"key": "auth-username",
+			"value": "TxTest",
+			"enabled": true
+		},
+		{
+			"key": "auth-password",
+			"value": "password",
+			"enabled": true
+		}
+	],
+	"_postman_variable_scope": "environment",
+	"_postman_exported_at": "2020-07-16T15:24:56.421Z",
+	"_postman_exported_using": "Postman/7.28.0"
+}
```

Comment:
You may want to configure your editor to add a newline at the end of files.

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file